docs/content/docs/dev/datastream/testing.md

```diff
@@ -449,26 +449,41 @@ public class ExampleIntegrationTest {
         CollectSink.values.clear();
 
         // create a stream of custom elements and apply transformations
-        env.fromElements(1L, 21L, 22L)
+        env.fromData(1L, 21L, 22L)
```

Comment:
This is difficult to review without trying it. It would be great if this class or a variant was in Flink as a test class, with a comment to say if the class is changed then change the docs. In this way we should have a working test and valid docs.